src/engine/engine.js

```diff
@@ -4,7 +4,7 @@ import { AudioListener } from "three";
 import EngineEditorCamera from "./util/cameracontrols/engineeditorcamera";
 import SessionHandler from "./util/webxr/sessionhandler";
 import Renderer from "./renderer";
-import Physics from "./physics/physics";
+// import Physics from "./physics/physics";
```

Comment:
Can the commented lines just be deleted?

Reply:
it can - worth noting that CannonJS is currently incompatible with our version of three and this is done to prevent it throwing errors. I'll add an issue